coordinator/src/services/translation/request/dispense/dispense-notification.ts

```diff
@@ -44,23 +41,25 @@ export async function convertDispenseNotification(
     fhirFirstMedicationDispense,
     fhirFirstMedicationDispense.performer[0].actor.reference,
   )
-  const fhirOrganisation = fhirContainedPractitionerRole.organization as fhir.IdentifierReference<fhir.Organization>
+  const fhirOrganisationRef = fhirContainedPractitionerRole.organization as fhir.Reference<fhir.Organization>
```

Comment:
rather than casting here we could typeguard this with an error:
```
if (!isReference(fhirContainedPractitionerRole.organization)) {
 throw new Error('give me reference...')
}
```